Library/Homebrew/test/test_cmd_audit.rb

```diff
@@ -156,6 +156,23 @@ class Foo < Formula
       fa.problems
   end
 
+  def test_audit_strict_resource_placement
```

Comment:
You might want to use `test_audit_file_strict_resource_placement` to match the other methods, i.e. `test_<method>_<details>`.